packages/api/src/env.ts

```diff
@@ -1,10 +1,17 @@
-import { z, createEnv, presetEnvOptions, nodeEnvSchema } from "@blobscan/zod";
+import {
+  z,
+  createEnv,
+  presetEnvOptions,
+  nodeEnvSchema,
+  booleanSchema,
+} from "@blobscan/zod";
 
 export const env = createEnv({
   envOptions: {
     server: {
       SECRET_KEY: z.string(),
       NODE_ENV: nodeEnvSchema.optional(),
+      METRICS_ENABLED: booleanSchema.default("false"),
```

Comment:
Current booleanSchema implementation already defaults to "false"

Reply:
No, it doesn't.

It will throw an error if the env var is not defined